packages/profile/src/components/inventory/collection/send/index.tsx

```diff
@@ -36,44 +42,97 @@ export function SendCollection() {
   const paramsTokenIds = searchParams.getAll("tokenIds");
   const { tokenId } = useParams<{ tokenId: string }>();
 
+  const { entrypoints } = useEntrypoints({
+    address: collectionAddress || "0x0",
+  });
   const { address } = useAccount();
   const { parent } = useConnection();
-  const [validated, setValidated] = useState(false);
-  const [warning, setWarning] = useState<string>();
+  const [recipientValidated, setRecipientValidated] = useState(false);
+  const [entrypointValidated, setEntrypointValidated] = useState(false);
+  const [recipientWarning, setRecipientWarning] = useState<string>();
+  const [entrypointWarning, setEntrypointWarning] = useState<string>();
```

Comment:
I would prepare object state like `type InputState =  {validated: boolean; warning?: string}` to avoid making too many states